packages/react-components/src/components/VideoTile.tsx

```diff
@@ -240,16 +239,15 @@ export const VideoTile = (props: VideoTileProps): JSX.Element => {
             background: theme.palette.neutralLighter,
             borderRadius: theme.effects.roundedCorner4
           },
+          isSpeaking
+            ? {
+                outlineOffset: '-0.25rem',
```

Comment:
didn't know about this property - nice one!

Reply:
I didn't even know this was possible